include/nanobind/nb_class.h

```diff
@@ -57,7 +57,12 @@ enum class type_flags : uint32_t {
     is_arithmetic            = (1 << 15),
 
     /// This type is an arithmetic enumeration
-    has_type_callback        = (1 << 16)
+    has_type_callback        = (1 << 16),
+
+    /// The user requested dynamic attributes. (i.e. a dict in the instance)
+    /// Note that the actual presence of an instance dict will depend on more
```

Comment:
Not sure what lines 2 and 3 of this comment add, the same could be said about all other flags. I'd delete them.